tasks/main.yml

```diff
@@ -63,6 +63,28 @@
     mode: u=rw,g=r
   register: koha_conf
 
+- name: Check for fi-FI translation
+  command: koha-translate --list
+  failed_when: False
+  register: koha_installed_translations
+  changed_when: '"fi-FI" not in koha_installed_translations.stdout'
+  when: koha_install_fi_translation
+
+- name: Install fi-FI translation
+  shell: "koha-translate --install fi-FI || koha-translate --update fi-FI"
+  when: koha_install_fi_translation and koha_installed_translations.changed
+  register: koha_lng_install
+  failed_when: 
+    # TODO: this is temporary workaround with stderr check below:
```

Comment:
We should check if this can be fixed in this role, by the looks of it it is possible.

Reply:
I made with by adding env variables MEMCACHED_SERVERS=127.0.0.1:11211 MEMCACHED_NAMESPACE=KOHA before **koha-translate** command invocation so it not warns now and passes ansible well.